sinatra-contrib/spec/reloader_spec.rb

```diff
@@ -429,15 +429,18 @@ def self.registered(klass)
     end
 
     it "allows block execution after reloading files" do
-      app_const.after_reload do
-        $reloaded = true
+      app_const.after_reload do |files|
+        $reloaded = files
       end
```

Comment:
My only concern is this code now executes this block with the `reloaded_paths` parameter, where it didn't before.

What's the behavior if that parameter is unused within user code?

Is this a breaking change that is already documented?

Thanks for my mostly random questions, as I haven't touched `after_reload` in a long time. 😅

Reply:
I will add some rspec to ensure this. I don't expect it to break any existing `after_reload` blocks out in the world.

Reply:
Hopefully 0074f9f7c4a32955d7522998909249a4ca3c2d00 addresses this.

Reply:
I guess it would break cases where `after_reload` is passed a lambda, like
```ruby
my_reload = -> { "This is a Proc with arity of 0!" }
# . . .
after_reload(&my_reload) # Instead of usual do...end
```
It's a judgment call whether you want to risk a break like this.

Reply:
Well I had some spare time tonight so I guarded against this issue too. See 7d3b4f404c853849f35ed03a586fade99d2dee7d.

I would vote that this change is proven non-breaking, at least pretty close to 100%.  :slightly_smiling_face: